tsconfig.json

```diff
@@ -12,7 +12,10 @@
     "noUnusedLocals": true,
     "jsx": "react",
     "moduleResolution": "node",
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "plugins": [
+      { "transform": "ts-optchain/transform" },
```

Comment:
![image](https://user-images.githubusercontent.com/13611391/66795550-241ea400-ef2f-11e9-9b20-91cb15499064.png)


This doesn't work yet (still use `oc` and not transpiling the code). To make it work I think you need to install [ttypescript](https://github.com/cevek/ttypescript), and move this transform config to webpack config instead (Not sure why it doesn't work when this config is in `tsconfig.json`).

Anyway, I'm fine if you just remove this part of code, and live without the transpiled code for now. We didn't support IE11 with the old `oc` anyway because it uses Proxies, and Proxies can't be polyfilled.

Also, I think we will migrate from `ts-loader` to `babel-loader` soon. So after that we could use [babel-plugin](https://github.com/epeli/babel-plugin-ts-optchain) for transpiling the code instead.

In summary either
1) make this `transform` plugin work, or
2) remove this `transform` plugin config for now

Reply:
I will check it again and make the appropriate changes.

Reply:
I used ttypescript and made the changes in webpack config in dev.js 
```js
{
        test: /\.tsx?$/,
        loader: 'ts-loader',
        options: {
          compiler: 'ttypescript'
        },
        exclude: /node_modules/
}
```
No change, i can see this code ```js exports.translationsSelector = function (state) { return ts_optchain_1.oc(state).settings.translations(); }; ``` and is not transpiled. 

Any guess why its not doing so ?

Reply:
Require `ts-optchain/transform`
```
const optChainTransform = require('ts-optchain/transform').default;
```

And try adding `getCustomTransformers` to `options` like so:
```
options: {
  compiler: 'ttypescript',
  getCustomTransformers: (program) => ({
    after: [optChainTransform(program)],
  })
}
```

Reply:
Thanks @paibamboo This works. I have pushed the changes for dev and prod.

Reply:
can you check also for server config? that's used for server side rendering